package.json

```diff
@@ -258,6 +258,8 @@
     "typescript": "^4.1.2",
     "unified": "^9.2.0",
     "unist-util-visit": "^2.0.3",
+    "utf-8-validate": "^5.0.3",
```

Comment:
`utf-8-validate` has marked as optional peer dependency in `ws` that Marp CLI is depending. It must not be required but `npm ls` throws an error while packaging through vsce.